plugins/modules/virt_cloud_instance.py

```diff
@@ -56,6 +56,15 @@
       - Only used when O(base_image) is a URL.
       - Useful for updating cached images to their latest versions.
     default: false
+  force_disk:
+    type: bool
+    description:
+      - Force deletion and recreation of existing disk files.
+      - When set to V(true), any existing disk files specified in O(disks) will be removed and recreated.
+      - When set to V(false), the module will fail if any disk file already exists.
+      - This parameter only affects disk file creation, not VM definition.
+      - Use with O(recreate=true) to recreate both the VM and its disk files.
+    default: false
```

Comment:
```suggestion
    default: false
    version_added: '2.1.0'
```

Reply:
Since `virt_cloud_instance` is a unreleased module, it seems that there's no need to add the `version_added` property here.